code/modules/mob/living/blood.dm

```diff
@@ -233,8 +233,8 @@
 	if(has_status_effect(/datum/status_effect/buff/lux_drained))
 		return LUX_DRAINED
 
-	if(istype(blood, /datum/blood_type/human/tiefling))
-		return LUX_FUCK_YOU_TIEFLING
+	// if(istype(blood, /datum/blood_type/human/tiefling))
+	// 	return LUX_FUCK_YOU_TIEFLING
```

Comment:
shadow tiefling buff ?

Reply:
Tiefling are under the inhumen meant to have Lux.